### ISSUE TITLE
Resize default annotation windows to 2/3 image & 1/3 text?

### ISSUE THREAD

**Issue by Quintman** [yale-web-technologies/mirador-project#89]
Right now the ToC brings a Mirador window split 50/50 between mural image and text annotation window. Can we tweak that to increase the default window sizes to prioritize the images, say 2/3 image and 1/3 text?

**Comment by gigamorph** [comment]
That feature had been deployed before you created this ticket, which means, inducing also from other problems you had from your machine, the browser cache often didn't get flushed. I have added some "cache-busting" techniques to the application. So please try if you still see the old problems. Thanks.

**Comment by Quintman** [comment]
Yes, thanks. We noticed it at our meeting with Lec last week. We also discussed it with him. I think we need to consult with 10KR about the final ratio. LOTB would prefer closer to 70% image 30% text.
